<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConcertInfoWidget</class>
 <widget class="QWidget" name="ConcertInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>433</width>
     <height>552</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::ExpandingFieldsGrow</enum>
    </property>
    <property name="rowWrapPolicy">
     <enum>QFormLayout::WrapLongRows</enum>
    </property>
    <property name="labelAlignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_files">
      <property name="text">
       <string>Files</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="files">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbl_title">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="title">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lbl_artist">
      <property name="text">
       <string>Artist</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="artist"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lbl_album">
      <property name="text">
       <string>Album</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="album"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lbl_tagline">
      <property name="text">
       <string>Tagline</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="tagline"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="lbl_rating">
      <property name="text">
       <string>Rating</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <widget class="QDoubleSpinBox" name="rating">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_userRating">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblUserRating">
        <property name="text">
         <string>User Rating</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="userRating">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="lbl_released">
      <property name="text">
       <string>Released</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDateEdit" name="released">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="displayFormat">
       <string notr="true">yyyy</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="lbl_runtime">
      <property name="text">
       <string>Runtime</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="runtime">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="suffix">
       <string> Minutes</string>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="lbl_certification">
      <property name="text">
       <string>Certification</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QComboBox" name="certification">
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContents</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="lbl_trailer">
      <property name="text">
       <string>Trailer</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="trailer"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="lbl_playCount">
      <property name="text">
       <string>Play Count</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="playcount">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_lastPlayed">
        <property name="text">
         <string>Last Played</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="lastPlayed">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Badge" name="badgeWatched">
        <property name="text">
         <string>Watched</string>
        </property>
        <property name="activeText" stdset="0">
         <string>Watched</string>
        </property>
        <property name="inactiveText" stdset="0">
         <string>Not watched</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="lbl_overview">
      <property name="text">
       <string>Overview</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QTextEdit" name="overview"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblIDs">
      <property name="text">
       <string>IDs</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayoutTmdb">
        <item row="0" column="0">
         <widget class="QLabel" name="lblTmdbId">
          <property name="text">
           <string>TMDb ID</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="tmdbId">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayoutImdb">
        <item row="0" column="1">
         <widget class="QLineEdit" name="imdbId">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lblImdbId">
          <property name="text">
           <string>IMDb ID</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblOriginalTitle">
      <property name="text">
       <string>Original Title</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="originalTitle"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Badge</class>
   <extends>QLabel</extends>
   <header>ui/small_widgets/Badge.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
